pom.xml

```diff
@@ -10,14 +10,138 @@
   <url>https://teragrep.com</url>
   <properties>
     <changelist>-SNAPSHOT</changelist>
+    <functionAppName>aer_02-20241015142323079</functionAppName>
     <java.version>11</java.version>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <metrics.version>4.2.8</metrics.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <prometheus-simpleclient.version>0.16.0</prometheus-simpleclient.version>
     <revision>0.0.1</revision>
-    <sha1></sha1>
```

Comment:
do not remove sha1, it is part of our pipeline scripts.

Reply:
added sha1 property to pom.xml